doc/source/projects.rst

```diff
@@ -0,0 +1,74 @@
+Ray Projects (Experimental)
+===========================
+
+Ray projects make it easy to package a Ray application so it can be
+rerun later in the same environment. They allow sharing and reuse
+existing code reliably.
+
+Quick start (CLI)
+-----------------
+
+.. code-block:: bash
+
+    # Creates a project in the current directory. It will create a
+    # project.yaml defining the code and environment and a cluster.yaml
+    # describing the cluster configuration.
+    $ ray project create <project-name>
+
+    # Create a new session from the given project with an optional name.
```

Comment:
What happens if you don't give it a name? autogen'd?

Reply:
At the moment it will be autogenerated as the project name, once we support multiple concurrent sessions we should have a smarter way to autogenerate it.